### DISCUSSION TITLE
Building Motion+ from source

### DISCUSSION THREAD

**Discussion by sroller** [Motion-Project/motionplus#107] [Q&A] [answered]
I started over with a fresh installation of Bookworm 64-bit on a Raspberry PI 3+.
I'd like to build with libcam enabled. `--with-libcam=DIR`
But whatever I try with a dir, without a dir at the end `libcamera support` show always up as `no`.
What do I do wrong?

**Comment by Mr-Dave** [comment, marked as answer]
I usually use the packaging tool to create a deb.  In that script, it includes the following packages `libcamera-tools` and `libcamera-dev` .  With those packages, it shouldn't be necessary to specify a dir.  (The guide package list does need to get updated to include these two)

**Reply by sroller** [reply]
I guess libcamera-dev is the key. I'll follow your recommendation wrt the packaging tool. You should know best!

**Comment by sroller** [comment]
I'm happy to report that I have (again) a running system. After installing libcamera-dev Motion+ can control my Pi camera (v2.1). What remains to do is to adjust the parameters and scripts to the new software.
The packaging also worked, but eventually I just run the famous
```
./configure
make
sudo make install
```
sequence which was successful.
